bin/lib/c-workflow.js

```diff
@@ -13,6 +13,10 @@ const workflows = [
         description: 'Use this to initialise the project (generally after \'git clone\').',
         name: 'project-init',
     },
+    {
+        description: 'Use this to re-initialise the project (generally after \'gco <pull-request-branch>\').',
+        name: 'pr-init',
```

Comment:
This can then be used to restart the project (not necessarily related to PRs)
```suggestion
        name: 'start',
```

Reply:
@allanchau, I think they're different things to be honest. And start might be good idea in itself. However, when you're checking out PRs, I generally like to do stop, build and start because it avoids potential issues when you don't rebuild all changed containers. I don't like the idea of including `c kc stop` in a start command, which is why I think they should be different.

The concept of workflow is to have to a few key processes that developers at idearium follow at logical points of the developer workflow. I think init, start and pr-init (or some other name) are all important ones really...

Reply:
what about `c workflow restart` or `c workflow pr`? I prefer `restart` though because sometimes you just want to trash everything and restart.